__tests__/app.test.js

```diff
@@ -51,6 +51,64 @@ describe("App", () => {
             });
         });
     });
+    describe("GET /api/articles", () => {
+        it("Check if endpoint returns an array of articles objects", () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.articles).toBeInstanceOf(Array);
+                expect(response.body.articles[0]).toEqual(expect.any(Object));
+            });
+        });
+        it("Check if body array of object matches objects in test data and its format", () => {
+            const firstArticle =   {
+                article_id : 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: 1594329060000,
+                votes: 100,
+                article_img_url:
+                  'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 11
+              }
+            const createdAtString = new Date(firstArticle.created_at - 3600000);
+            firstArticle.created_at = createdAtString.toISOString();
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.articles).toHaveLength(12);
+                response.body.articles.forEach((articleObj) => {
+                    if (articleObj.article_id === 1) expect(articleObj).toEqual(expect.objectContaining(firstArticle));
+                    expect(articleObj).toMatchObject( {
+                        article_id : expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(Number)
+                      }
+                    );
+                });
+            });
+        });
+        it("Check if body array of objects are sorted by date and in descending order", () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const testDataExpectedOrder = [3, 6, 2, 12, 5, 1, 9, 10, 4, 8, 11, 7]
+                const testDataActualOrder = body.articles.map(articleObj => articleObj.article_id)
+                expect(testDataExpectedOrder).toEqual(testDataActualOrder); 
```

Comment:
This would be a good chance to use jest-sorted for this test. (just a suggestion, you don't need to change this but it will make your code cleaner) Here is a link below:

https://www.npmjs.com/package/jest-sorted